app/cmd/root.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	httpProcessTimeout time.Duration
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "boilerplate-go",
```

Comment:
better use `goboilerplate` for easier replacement on CLI app
```suggestion
	Use:   "goboilerplate",
```